sysutils/git-chglog/Portfile

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/git-chglog/git-chglog 0.9.1
+revision            0
+
+categories          sysutils
+platforms           darwin
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+description         git CHANGELOG generator
+long_description    ${description}. Anytime, anywhere, Write your CHANGELOG
+homepage            https://github.com/git-chglog/git-chglog
```

Comment:
Hey @judaew, thanks for this submission.  One comment though; you should not need to specify the homepage.  It should be set automatically since you are sourcing from `github.com` and you are using the `golang` portgroup.